src/components/author/author.stories.mdx

```diff
@@ -56,6 +56,16 @@ Displays an author, or set of authors, for article bylines.
 - `class`: Append a class to the root element.
 - `authors`: Array of [author objects](https://timber.github.io/docs/reference/timber-user/#properties) of the type:
 
+  ```ts
+  {
+    name: string;
```

Comment:
@gerardo-rodriguez Would it be more clear to have the "or" inline?

```
    name: string | () => string;
```

Reply:
Thanks, @calebeby!

https://github.com/cloudfour/cloudfour.com-patterns/pull/1300/commits/9778e03e064c88a036730f63cc660780c6b5abbd

Can you approve the PR one more time so I can merge? Thank you! 🙂